### PR TITLE
math-magicians-app-setup

### PR DESCRIPTION
**Pull request by Shaheryar0054**
- Created react app from given instructions.
- Added linters to check any styling or JS errors.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @Shaheryar0054, Kindly use the updated version  of the microverse [readme](https://github.com/microverseinc/readme-template) template for your readme, all sections from the microverse template should be present in your readme, only the FAQ is optional

**Comment by prolajumokeoni** [comment]
Hi @Shaheryar0054 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional readme
Kindly use the updated version  of the microverse [readme](https://github.com/microverseinc/readme-template) template for your readme


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [React-app] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[React-app]** In this project we build simple application that have ability to add and remove books
+
```
Comment (julie-ify) [inline README.md]: - [ ] Kindly fix your project title and description.

- [ ] Kindly remove the text from the template, "Describe your project in 1 or 2 sentences.". This is for demo purposes and should not be a part of the update readme.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
```
Comment (julie-ify) [inline README.md]: - [ ] Kindly remove text from template.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  </ul>
+</details>
+<ul>
+    <li><a href="https://reactjs.org/">React</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
Comment (julie-ify) [inline README.md]: - [ ] Kindly remove text from template.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+- **[DOM manipulating]**
+- **[Setup-the-basic-app]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
```
Comment (julie-ify) [inline README.md]: - [ ] Kindly remove text from template.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
```
Comment (julie-ify) [inline README.md]: - [ ] Kindly remove text from template.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+- GitHub: [@shaheryar0054](https://github.com/Shaheryar0054)
+- Twitter: [@sharya0310](https://twitter.com/home)
+- LinkedIn: [@shaheryar](https://www.linkedin.com/in/shaheryar-abid-8758121b3/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```
Comment (julie-ify) [inline README.md]: - [ ] Kindly remove text from template.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Shaheryar0054,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @Shaheryar0054,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**